### PR TITLE
Header refactor

### PR DESCRIPTION
**Pull request by oliverlloyd**
## What does this change?
This PR refactors `Header` to reduce the size of the logo.

I took the chance here to consider the [larger use case for a Header at the Guardian](https://github.com/guardian/source/discussions/816) and changed things with this in mind. Perhaps a follow on PR from this could be to hoist the `Logo` component into Source? I looked at how Header and Logo are implemented in DCR and made sure to consider their use cases as well so hopefully any Source version will be useful in other places.

## `logoType`
Allows the version of the logo to be specified
```type LogoType = 'standard' | 'anniversary' | 'bestWebsite';```

See: https://github.com/guardian/source/pull/935 for more details

https://user-images.githubusercontent.com/1336821/130223316-d660384f-c9a4-49c0-beb9-8829c6a1f962.mov

### CODE REVIEW

File: src/client/components/Logo.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import { Link } from '@guardian/src-link';
+import { from } from '@guardian/src-foundations/mq';
+import { visuallyHidden } from '@guardian/src-foundations/accessibility';
+
+const svgSize = css`
+  svg {
+    width: 149px;
+    height: 50px;
+    ${from.tablet} {
+      width: 200px;
```
Comment (ob6160) [inline src/client/components/Logo.tsx:15]: how are these sizes at each breakpoint level decided? are they done by eye or set out in DCR?

File: src/client/components/Logo.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+export const Logo = ({ isAnniversary }: { isAnniversary?: boolean }) => (
+  <Link
+    href="https://www.theguardian.com"
+    title="The Guardian Homepage"
+    subdued={true}
+    cssOverrides={css`
+      color: ${brandText.primary};
+      :hover {
+        color: ${brandText.primary};
```
Comment (ob6160) [inline src/client/components/Logo.tsx:74]: just wondering why have this hover selector if it doesn't change the colour

**Review by ob6160 — COMMENTED** [review]
looks good :) just a few comments

File: src/client/components/Logo.tsx (hunk 1 above)
Comment (oliverlloyd) [reply src/client/components/Logo.tsx:15]: Yeah, that's a really good point. We should align with DCR, let me look into doing that.

File: src/client/components/Logo.tsx (hunk 2 above)
Comment (oliverlloyd) [reply src/client/components/Logo.tsx:74]: This is there to override the hover value that you get when using `Link`

Comment (ob6160) [reply src/client/components/Logo.tsx:74]: ah yes, missed that :)

File: src/client/components/Logo.tsx (hunk 1 above)
Comment (ob6160) [reply src/client/components/Logo.tsx:15]: cool cool

Comment (oliverlloyd) [reply src/client/components/Logo.tsx]: This is now done. We should be using the same exact sizes as DCR

**Review by ob6160 — APPROVED** [review]
🥇 nice one

File: src/client/components/Logo.tsx (hunk 1 above)
Comment (ob6160) [reply src/client/components/Logo.tsx]: brill :)

**Comment by oliverlloyd** [comment]
[Forgotten password: Reduce logo size and margins to match dotcom](https://trello.com/c/IGr9x5Td/3289-forgotten-password-reduce-logo-size-and-margins-to-match-dotcom)

**Pull request merged**